composer.json

```diff
@@ -4,10 +4,6 @@
 	"homepage": "https://github.com/wmde/PsrLogTestDoubles",
 	"license": "GPL-2.0-or-later",
 	"authors": [
-		{
-			"name": "Wikimedia Deutschland FUN Team",
-			"email": "fundraising-tech-internal@wikimedia.de"
-		},
 		{
 			"name": "Jeroen De Dauw",
 			"email": "jeroendedauw@gmail.com",
```

Comment:
what's the reason for removing WMDE but keeping Jeroen even though this is a repo under the `wmde` namespace on gitlab?